docs/iauditor-exporter_configure.md

```diff
@@ -23,6 +23,7 @@ iauditor-exporter configure [flags]
       --inspection-completed string         Return completed inspections, false, true or both (default "both")
       --inspection-include-inactive-items   Include inactive items in the inspection_items table (default false)
       --inspection-incremental-update       Update inspections, inspection_items and templates tables incrementally (default true)
+      --inspection-modified-after string    Return inspections modified after this date
```

Comment:
Does these API's support all formats of timestamps ?
If not, can you add the supported timestamps.